app/src/androidTest/java/org/mozilla/fenix/ui/robots/SearchRobot.kt

```diff
@@ -534,11 +534,12 @@ private fun assertTranslatedFocusedNavigationToolbar(toolbarHintString: String)
         ).waitForExists(waitingTime),
     )
 
-private fun assertFillLinkButton() {
-    mDevice.waitForIdle()
-    onView(withId(R.id.fill_link_from_clipboard))
-        .check(matches(withEffectiveVisibility(ViewMatchers.Visibility.GONE)))
-}
+private fun assertFillLinkFromClipboardIsNotDisplayed() =
```

Comment:
For the future: Let's avoid doing duplicate code like this and skip the assert... methods (to keep the code cleaner and the classes shorter).

Reply:
Done !